### PR TITLE
Add some tests to Value

### PR DESCRIPTION
**Pull request by gnieto**
Added tests for reading Value for Byte, Short, Char, Int, Long, Float and Double. I'll add missing tests on a further PR

### CODE REVIEW

**Comment by Razican** [comment]
We really need tests, yes. Also, we need to run cargo fmt, could you? (with rustfmt 0.8.1) and check the new minimum version? seems that it's not compiling in Rust 1.13.

**Comment by gnieto** [comment]
Checking it!

File: src/types/read.rs [hunk 1]
```diff
@@ -274,24 +274,24 @@ impl Visibility {
...
-const VALUE_STRING: u8 = 0x17;
-const VALUE_TYPE: u8 = 0x18;
-const VALUE_FIELD: u8 = 0x19;
-const VALUE_METHOD: u8 = 0x1a;
-const VALUE_ENUM: u8 = 0x1b;
-const VALUE_ARRAY: u8 = 0x1c;
-const VALUE_ANNOTATION: u8 = 0x1d;
-const VALUE_NULL: u8 = 0x1e;
-const VALUE_BOOLEAN: u8 = 0x1f;
+pub const VALUE_BYTE: u8 = 0x00;
```
Comment (gnieto) [inline src/types/read.rs]: This is need for Rust <1.15
I don't really know why this does not work on that versions.

Comment (Razican) [reply src/types/read.rs]: If it works on stable, I prefer it to be private. (we might need to push the minimum Rust version on Travis to 1.15.1, but I'm fine with it)

**Comment by Razican** [comment]
Appveyor versions also need to be updated

**Pull request merged**